source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -193,6 +193,19 @@ If your request included the `state` parameter, the URI will also include this p
 
 ### Error handling for ‘Make a request to the `/authorize` endpoint’
 
+You must check the HTTP return code from   the `/authorize` request.
+
+#### HTTP 502 Bad Gateway
+If you receive `{"message": "Internal server error"}` with an HTTP 502 (Bad Gateway) in response to your response `GET` request to the `/authorize` endpoint you should check the [parameters are correct][integrate.replace-example-message].
```

Comment:
```suggestion
If your response `GET` request to the `/authorize` endpoint produces this error: `{"message": "Internal server error"}` with an HTTP 502 (Bad Gateway), you should check the [parameters are correct][integrate.replace-example-message].
``` 

Does this keep the same meaning? I have flipped it around a bit to make it a bit clearer. Also, is it definitely a response `GET` request and not just a `GET` request? Just wanted to check as it sounds odd but probably just as it's not something I have come across yet!

Reply:
```suggestion
If you receive `{"message": "Internal server error"}` with an HTTP 502 (Bad Gateway) in response to your `GET` request to the `/authorize` endpoint you should check the [parameters are correct][integrate.replace-example-message].
```

Reply:
that reads much better, it was probably what I meant to say all along :) appreciated @ImogenCraigmile